R/mod_utilization.R

```diff
@@ -0,0 +1,151 @@
+#' utilization UI Function
+#'
+#' @description A shiny Module.
+#'
+#' @param id,input,output,session Internal parameters for {shiny}.
+#'
+#' @noRd 
+#'
+#' @importFrom shiny NS tagList 
+# mod_UI
+mod_utilization_ui <- function(id){
+  ns <- shiny::NS(id)
+  shiny::fluidPage(
+    ui_header_row(),
+    ui_row(box(
+      pickerInput(
+        inputId = "providerListUtilization",
```

Comment:
If the above is used, this will automatically become `program` and the `ns` function will be applied. The `ns` function must always be applied to inputIds in modules, but the `ui_picker_program` function does it by default.